src/core/evaluator.js

```diff
@@ -953,8 +953,9 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
           });
       }
 
-      function buildTextGeometry(chars, textChunk) {
+      function buildTextGeometry(chars, textChunk, previousChars) {
```

Comment:
Is there a way to handle previousChar in the if below? It does not look logical handle that in two places